src/common/topology.js

```diff
@@ -574,6 +574,10 @@ var TopologyMap = function($http, topology) {
         .duration(500)
         .attr('r', r)
         .attr('_r', r);
+    })
```

Comment:
this section gets called and applied to all nodes on every update, and its already setting the transform, so I think this would be the more appropriate place to update the scale as well.

https://github.com/ripple/ripplecharts-frontend/blob/develop/src/common/topology.js#L533-L544